sklearn/cluster/tests/test_k_means.py

```diff
@@ -238,29 +333,10 @@ def test_k_means_new_centers():
 
 
 @if_safe_multiprocessing_with_blas
-def test_k_means_plus_plus_init_2_jobs():
-    km = KMeans(init="k-means++", n_clusters=n_clusters, n_jobs=2,
-                random_state=42).fit(X)
-    _check_fitted_model(km)
-
-
-def test_k_means_precompute_distances_flag():
-    # check that a warning is raised if the precompute_distances flag is not
-    # supported
-    km = KMeans(precompute_distances="wrong")
-    with pytest.raises(ValueError):
-        km.fit(X)
-
-
-def test_k_means_plus_plus_init_not_precomputed():
-    km = KMeans(init="k-means++", n_clusters=n_clusters, random_state=42,
-                precompute_distances=False).fit(X)
-    _check_fitted_model(km)
-
-
-def test_k_means_random_init_not_precomputed():
-    km = KMeans(init="random", n_clusters=n_clusters, random_state=42,
-                precompute_distances=False).fit(X)
+def test_k_means_plus_plus_init_2_threads():
```

Comment:
Trying to understand, is there anything special to the combination k-means++ with 2 threads/jobs?

Reply:
Maybe in the previous version where you'd do the init in parallel. Now that the parallel part has moved inside each run, there's absolutely no interaction between init and n_jobs. I guess I can simply remove the test.